libavcodec/hevc_filter.c

```diff
@@ -712,7 +712,7 @@ static int boundary_strength(HEVCContext *s, MvField *curr, MvField *neigh,
 }
 
 void ff_hevc_deblocking_boundary_strengths(HEVCContext *s, int x0, int y0,
-                                           int log2_trafo_size)
```

Comment:
we have to do a second pass of this now after the image is decoded, for coding tree units (encoded blocks) that are on the edge of a tile, but only for the outside - so this new flag for "do_internal"